db/migrate/20250807120146_create_establishments.rb

```diff
@@ -0,0 +1,53 @@
+class CreateEstablishments < ActiveRecord::Migration[7.2]
+  def change
+    # rubocop:disable Metrics/BlockLength
+    create_table(:lists_establishments) do |t|
```

Comment:
think you could put the rubocop:disable on this line then it would just apply to the block and the rubocop:enable wouldn't be needed